susemanager-utils/cluster-providers/caasp/uyuni-cluster-provider-caasp.spec

```diff
@@ -0,0 +1,73 @@
+#
+# spec file for package uyuni-cluster-provider-caasp
+#
+# Copyright (c) 2020 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+%if 0%{?suse_version} > 1320
```

Comment:
If you don't care about the differences between SLE and openSUSE, maybe you can consider:
` %if 0%{?sle_version} >= 15000`

Reply:
done